src/components/PageHero.tsx

```diff
@@ -48,113 +49,93 @@ const PageHero = ({
   children,
   className,
 }: PageHeroProps) => (
-  <Box py={4} px={8} width="full">
+  <div className="w-full px-8 py-4">
     <Flex
-      justifyContent="space-between"
-      mt={8}
-      px={{ base: 0, lg: 16 }}
-      direction={{ base: isReverse ? "column" : "column-reverse", lg: "row" }}
-      className={className}
+      className={cn(
+        "mt-8 justify-between px-0 lg:px-16",
+        isReverse ? "flex-col" : "flex-col-reverse",
+        "lg:flex-row",
+        className
+      )}
     >
-      <Box
-        maxW={{ base: "full", lg: "container.sm" }}
-        pt={{ base: isReverse ? 0 : 8, lg: 32 }}
-        pb={{ base: isReverse ? 8 : 0, lg: 32 }}
-        ps={{ base: 0, lg: 8 }}
-        me={{ base: 0, lg: 4 }}
+      <div
+        // TODO: container.sm width unavailable
```

Comment:
Removing comment since we will not have this measure (640px) on the new theme. We can treat it now as a custom size.

Reply:
Gotcha!! So every `container.sm` is considered as a custom size of `(640px)` .

Reply:
Yeah, we don't have that breakpoint on the theme anymore.